thirdeye/thirdeye-frontend/app/styles/shared/_styles.scss

```diff
@@ -273,11 +273,16 @@ body {
   }
 }
 
-.te-alert-page-hzlist {
-    display: inline-block;
-    margin-left: 10px;
-    font-size: 15px;
-    padding: 0;
+.te-pill-selectors {
+  display: inline-block;
+  margin-left: 10px;
+  font-size: 15px;
+  padding: 0;
+
+  &__wrapper {
```

Comment:
`wrapper` shouldn't be nested if it's indeed a parent block

Reply:
done